cluster/charts/universal-crossplane/templates/crossplane/deployment.yaml

```diff
@@ -46,6 +46,9 @@ spec:
           args:
           - core
           - init
+          {{- if .Values.maxReconcileRate }}
+          - --max-reconcile-rate={{ .Values.maxReconcileRate }}
```

Comment:
Should we add this to values.yaml.tmpl ? 

https://github.com/crossplane/crossplane/blame/master/cluster/charts/crossplane/values.yaml.tmpl#L27

Reply:
Added, thanks for catching!